modules/web/src/app/node-data/component.ts

```diff
@@ -139,7 +139,7 @@ export class NodeDataComponent extends BaseFormValidator implements OnInit, OnDe
       ]),
       [Controls.Count]: this._builder.control(this._nodeDataService.nodeData.count),
       [Controls.DynamicConfig]: this._builder.control(this._nodeDataService.nodeData.dynamicConfig),
-      [Controls.OperatingSystem]: this._builder.control(this._getDefaultOS(), [Validators.required]),
+      [Controls.OperatingSystem]: this._builder.control({}, [Validators.required]),
```

Comment:
Can we not use an `{}`; declare an object and instead start with a falsy value like null?